.golangci.yml

```diff
@@ -0,0 +1,26 @@
+output:
+  sort-results: true
+
+run:
+  build-tags:
+  - withebpf
+  - tools
+  - docs
+  skip-files:
+  # Skip autogenerated files.
+  - ^.*.pb.go$
+
+issues:
+  exclude-use-default: false
+  max-same-issues: 0
+  max-issues-per-linter: 0
+  exclude-rules:
+  # Ignore autogenerated code.
+  - linters:
```

Comment:
Inconsistent indentation?

Reply:
How come? If you prefer indented lists, I can change it :)